isis/tests/FunctionalTestsJigsaw.cpp

```diff
@@ -143,4 +145,95 @@ TEST_F(ObservationPair, FunctionalTestJigsawCamSolveAll) {
   // DEC(t3) final
   EXPECT_NEAR(elems.at(56).toDouble(), 0.365717165,          0.00001); 
   
-}
\ No newline at end of file
+}
+
+TEST_F(ObservationPair, FunctionalTestJigsawErrorNoSolve) {
+  QTemporaryDir prefix;
+  QString outCnetFileName = prefix.path() + "/outTemp.net";
+  QVector<QString> args = {"fromlist="+cubeListFile, "cnet="+cnetPath, "onet="+outCnetFileName, 
+                           "camsolve=None", "spsolve=None"};
+
+  UserInterface options(APP_XML, args);
+  
+  Pvl log; 
+  
+  try {
+    jigsaw(options, &log);
+    FAIL() << "Should throw" << std::endl;
+  }
+  catch (IException &e) {
+    EXPECT_THAT(e.what(), HasSubstr("Must either solve for camera pointing or spacecraft position"));
+  }
+}
+
+
+TEST_F(ObservationPair, FunctionalTestJigsawErrorNoNet) {
+  QVector<QString> args = {"fromlist="+cubeListFile, "cnet=lolfake.net", "onet=doesnotmatter"};
+
+  UserInterface options(APP_XML, args);
+  
+  Pvl log; 
+  
+  try {
+    jigsaw(options, &log);
+    FAIL() << "Should throw an exception" << std::endl;
+  }
+  catch (IException &e) {
+    EXPECT_THAT(e.what(), HasSubstr("Unable to open"));
+  }
+}
```

Comment:
This doesn't need to be tested. We already test that ControlNet's read from a file method errors properly. Jigsaw isn't re-throwing or anything so this is just re-testing that.